### PR TITLE
feat: add push provider fields

### PR DESCRIPTION
**Pull request by peterdeme**
Add push provider fields

### CODE REVIEW

File: app.go [hunk 1]
```diff
@@ -238,6 +240,7 @@ type CheckPushResponse struct {
 	DeviceErrors             map[string]DeviceError `json:"device_errors"`
 	GeneralErrors            []string               `json:"general_errors"`
 	SkipDevices              *bool                  `json:"skip_devices"`
+	ProviderName             string                 `json:"provider_name"`
```
Comment (ferhatelmas) [inline app.go:243]: this is in wrong place, will be on `DeviceError` since each device can have a different provider

Comment (peterdeme) [reply app.go]: you're right! fixed

**Review by ferhatelmas — DISMISSED** [review]
LGTM

**Comment by peterdeme** [comment]
Rebased to master to solve merge conflict

**Review by ferhatelmas — APPROVED** [review]

**Pull request merged**